package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "author": "Octopus Deploy",
+  "bugs": {
+    "url": "https://github.com/OctopusDeploy/deploy-release-untenanted-action/issues"
+  },
+  "dependencies": {
+    "@actions/core": "^1.10.0",
+    "@octopusdeploy/api-client": "github:OctopusDeploy/api-client.ts#enh-cleanoutv1"
```

Comment:
this should change to a version pointing to a valid version hosted on npm before merging.